tests/integration/utils.sh

```diff
@@ -168,10 +167,24 @@ function container_run()
   fi
 }
 
+# Execute a command within a container.
+#
+# @container_options   Additional options for the 'podman container exec' command.
+# @container_name      The name or ID of the target container.
+# @container_command   The command to be executed within the container.
 function container_exec()
 {
-  # shellcheck disable=SC2068
-  podman container exec $@ 2> /dev/null
+  local container_options="$1"
+  local container_name="$2"
+  local container_command="$3"
```

Comment:
I think that if we reorder the arguments to

1. `container_name`
2. `container_command`
3. `container_options`

as `container_options` is optional and will be less used, the callers won't have to pass `''` when not using the `podman container exec` options, making the signature of the function cleaner (?)

I see this as a style decision, so feel free to stick to the way it is if you think it is better :grimacing:

Reply:
I just did this :)